src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <main>
+
+      <container class="stars stars--0">
```

Comment:
```suggestion
    <main>
      <div class="stars stars--0">
```
don't use custom tags